cli_tools/gce_vm_image_import/main.go

```diff
@@ -141,10 +148,32 @@ func validateAndParseFlags() error {
 			return err
 		}
 	}
+
 	return nil
 }
 
-//Returns main workflow and translate workflow paths (if any)
+// Validate source file is not a compression file by checking file header.
+func validateSourceFile(storageClient commondomain.StorageClientInterface) error {
+	if sourceFile == nil {
+		return nil
+	}
+
+	rc, err := storageClient.GetObjectReader(*sourceBucketName, *sourceObjectName)
+	if err != nil {
+		return fmt.Errorf("readFile: unable to open file from bucket %q, file %q: %v", *sourceBucketName, *sourceObjectName, err)
+	}
+	defer rc.Close()
+
+	// Try to extract compression file header of the source file.
+	_, err = gzip.NewReader(rc)
+	if err == nil {
+		return fmt.Errorf("source file is a compressed file. Please use 'images create' instead of 'images import'")
```

Comment:
Create image is not an alternative to image import in most cases (can't read most formats, can't do translate), so it's not a good idea to suggest it as such. The error message should only describe what is the problem and how to resolve it. Something like:

"cannot import an image from a compressed file. Please provide a path to an uncompressed image file"

Reply:
Sounds good. Still want to let people know that it's possible that they used wrong command, because that is why the bug is filed - they should have using 'image create'.
Let's say: "cannot import an image from a compressed file. Please provide a path to an uncompressed image file. If the compressed file is a exported image, please use 'image icreate' instead."

Reply:
... "If the compressed file is an image exported from Google Compute Engine, please use 'images create' instead."

Reply:
Great!